packages/user/Fractals/service/src/lib.rs

```diff
@@ -358,6 +350,15 @@ pub mod service {
         Guild::add(fractal, guild_account, get_sender(), display_name);
     }
 
+    /// Exile a fractal member.
+    ///
+    /// # Arguments
+    /// * `member` - The fractal member to be exiled.
+    #[action]
+    fn exile_member(member: AccountNumber) {
```

Comment:
As mentioned in slack, 

>  I think the sender should be the legislative account. And the action could also just take the fractal as a param, so it can just have (something like) `assert(getSender() == FractalTable::get(fractal).legislature)`. I don't think you want to look up the fractal based on the sender, otherwise you will need a another index on the fractal table.